src/PlaywrightSharp/BrowserContextOptions.cs

```diff
@@ -236,6 +254,21 @@ public BrowserContextOptions Clone()
                 args["recordVideo"] = RecordVideo;
             }
 
+            if (Proxy != null)
+            {
+                args["proxy"] = Proxy;
+            }
+
+            if (!string.IsNullOrEmpty(StorageStatePath))
+            {
+                StorageState = JsonSerializer.Deserialize<StorageState>(File.ReadAllText(StorageStatePath), JsonExtensions.DefaultJsonSerializerOptions);
```

Comment:
Same comment as above applies. Should we be more defensive here? Or do we just "ignore it" and let the use handle it?